added documentation of changes for PR #11769 (#11916)
mne-tools/mne-python: doc/changes/devel.rst

- Improve tests for saving splits with `Epochs` (:gh:`11884` by `Dmitrii Altukhov`_) 
- Added functionality for linking interactive figures together, such that changing one figure will affect another, see :ref:`tut-ui-events` and :mod:`mne.viz.ui_events` (:gh:`11685` by `Marijn van Vliet`_)
- HTML anchors for :class:`mne.Report` now reflect the ``section-title`` of the report items rather than using a global incrementor ``global-N`` (:gh:`11890` by `Eric Larson`_)

> - Added option to perform DC removal when computing Welch or multitaper spectra estimations. New option: remove_DC, True by default. Used in functions compute_psd for raw, Epoch and Evoked objects (:gh:`11769` by `Nikolai Chapochnikov`_)

Reviewer: ```suggestion
- Added option ``remove_dc`` to to :meth`Raw.compute_psd() <mne.io.Raw.compute_psd>`, :meth`Epochs.compute_psd() <mne.Epochs.compute_psd>`, and :meth`Evoked.compute_psd() <mne.Evoked.compute_psd>`, to allow skipping DC removal when computing Welch or multitaper spectra (:gh:`11769` by `Nikolai Chapochnikov`_)
```

> - Added option ``remove_dc`` to to :meth`Raw.compute_psd() <mne.io.Raw.compute_psd>`, :meth`Epochs.compute_psd() <mne.Epochs.compute_psd>`, and :meth`Evoked.compute_psd() <mne.Evoked.compute_psd>`, to allow skipping DC removal when computing Welch or multitaper spectra (:gh:`11769` by `Nikolai Chapochnikov`_)

Reviewer: ```suggestion
- Added option ``remove_dc`` to to :meth:`Raw.compute_psd() <mne.io.Raw.compute_psd>`, :meth:`Epochs.compute_psd() <mne.Epochs.compute_psd>`, and :meth:`Evoked.compute_psd() <mne.Evoked.compute_psd>`, to allow skipping DC removal when computing Welch or multitaper spectra (:gh:`11769` by `Nikolai Chapochnikov`_)
```
